sp800-63a/sec6_dc.md

```diff
@@ -2,64 +2,46 @@
+
+The following requirements detail how a CSP should validate the existence of a past proofing event (i.e. the credential) prior to issuance of a derived authenticator.  It also lists lifecycle management requirements to keep derived authenticators in sync with the primary authenticator.
+
+## 6.2. General Requirements
+
+1. The IAL of the credential bound to the derived authenticator SHALL be at or below the primary IAL. In most cases the IALs are expected to be the same in order to benefit from the reuse of the original identity proofing event(s). 
 1. Before issuance, the CSP SHALL verify the original authenticator status. The CSP SHALL NOT issue a derived authenticator if status indicates any type of termination, disablement, revocation, or expiration.
 2. Before issuance, the CSP SHALL verify that the corresponding authenticator is possessed and controlled by the claimant.  
 3. The derived authenticator SHALL be valid only as long as the subscriber is authorized to hold the original authenticator.
 4. The CSP SHALL record the details of the original authenticator used as the basis for derived authenticator issuance. 
-5. The CSP SHOULD set the expiration of the derived authenticator to the expiration of the primary authenticator. There are instances where the derived authenticator need not be directly tied to the expiration of the primary authenticator as the derived authenticator can provide authentication services in its place, for example, while the expiring primary credential is being replaced.
-6. The derived authenticator SHOULD be issued at an AAL equivalent to  the IAL the primary authenticator is bound to.  The derived authenticator MAY be issued at an AAL higher than the IAL the primary authenticator is bound to. For example, if the primary IAL is 2, the authenticator that is issued could be AAL2 or 3. The IAL the derived authenticator is bound to SHALL remain at the IAL of the primary authenticator.
+5. The CSP SHOULD set the expiration of the derived authenticator to the expiration, if any, of the primary authenticator. There are instances where the derived authenticator need not be directly tied to the expiration of the primary authenticator as the derived authenticator can provide authentication services in its place, for example, while the expiring primary credential is being replaced.
+6. The derived authenticator type MAY be any AAL, regardless of the AAL of the primary authenticator or the IAL of the bound credential.  
 
 
-### 6.1.2. IAL 2 Requirements
+## 6.3. AAL 2 Requirements
 
 - The CSP SHOULD check the status of the original authenticator weekly. 
 
 
-### <a name="dc-ial3"></a>6.1.3. IAL 3 Requirements
-
+## <a name="dc-ial3"></a>6.4. AAL 3 Requirements
+2. At IAL3, the CSP SHOULD verify in-person that a claimant possesses, controls and can successfully authenticate using the primary authenticator(s). 
```

Comment:
AAL3.  Good catch.  SHOULD because there is no reason in-person should be mandatory. i already proofed in person so why do it again.  This was confirmed with HIldy.  BTW - 800-157 does require in-person as a hardened profile on 800-63, but we shouldn't require it here.